### ISSUE TITLE
Infinite mana

### ISSUE THREAD

**Issue by FortyTwoCn** [ramidzkh/Applied-Botanics#71]
![2024-08-06_12 28 16](https://github.com/user-attachments/assets/5d8f8006-0bdf-42ef-8bef-8b1ca41b044a)
When the AE2 input bus is connected to the mana spreader， it can achieve Infinite mana
Botania-1.18.2-435
Applied-Botanics-1.0.3
appliedenergistics2-forge-11.7.6
1.18.2Forge 40.2.21

**Comment by ramidzkh** [comment]
Seems like a Botania bug in this version, where the mana spreader allows for negative mana but is just clamped above
```java
    public void receiveMana(int mana) {
        this.mana = Math.min(this.mana + mana, this.getMaxMana());
        this.setChanged();
    }
```

**Comment by ramidzkh** [comment]
Manually released for 1.18.2
